resources/definitions/Infinity3D.def.json

```
@@ -0,0 +1,77 @@
+{
+    "version": 1,
```

```suggestion
    "version": 2,
```
THis needs to be 2, as it defines the version of the format used.